bin/modules-docs.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+var https = require('https');
+var logger = require('console');
+
+var docs = path.join(__dirname, '..', 'docs', 'modules');
+var moduleReadmes = {
+    'geval': ['Raynos/geval', '2.1.1', 'README.md'],
```

Comment:
The idea is opposite: they *should be equal* to versions in `package.json`. Agree? 

Ho to do it?